project/Game.py

```diff
@@ -149,8 +149,43 @@ def start(self):
     def end(self):
         self.ended = True
 
-    def quit_question(self, team, password):
-        return True
+    def quit_question(self, dtn , team, password):
+        if(team.password==password):
+            team.currentLandmark+=1
+            if(dtn<team.clueTime):
+                team.timelog(datetime.timedelta(days=0,hours=0,minutes=0,seconds=0))
```

Comment:
timeLog is not a method, so what is this line doing? Was this supposed to be append?